lib/boilerPlate.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const util = require('util');
+
+/**
+ * Error for whatever you want!
+ *
+ * @extends Error
+ * @class Errors.BoilerPlate
+ * */
+class BoilerPlate extends Error{
+    /**
+ * @example
+
+        put your example here
+
+
+ * @example
+ *
+ * ```sh
+ *
+        and the output here
+
+ * ```
+ *
+ *
+ */
+    setMessage(param1, param2, scope){
+        this.name = 'New Error';
```

Comment:
This is very bad. IT does not let the user know that the name should match the class name.